base/services/w32time/w32time.h

```diff
@@ -16,6 +16,7 @@
 
 #define MAX_VALUE_NAME 16383
 #define NTPPORT 123
+#define W32TIME_POLL_INTERVAL 4 * 60 * 60 // 4 Hours
```

Comment:
```suggestion
#define W32TIME_POLL_INTERVAL (4 * 60 * 60) // 4 hours.
```
I would even call it `W32TIME_POLL_INTERVAL_HACK`.

Reply:
I say "It is not a HACK.", but "It is a PREFERENCE." Thanks.

Reply:
Changed as suggested. Thanks.